src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnection.cs

```diff
@@ -295,11 +295,21 @@ private void ConsumeFromRemainingBuffer(int bytesToConsume)
             {
                 Debug.Assert(_pool.UsingProxy);
 
+                // TODO: #28863 Part 1: Uri.IdnHost should include [] around IPv6 address.
```

Comment:
I would simplify these comments (which are repeated in a few places in this PR).

I would simply say:

```c#
// TODO: #28863 Uri.IdnHost is missing '[', ']' characters around IPv6 address. So, we need to add them manually for now.
```

I would remove all the text in the comments about Uri.Host.  It's not necessary to talk about that here because the handler should only be using .IdnHost anyways.  The bug with Uri.Host not including %number for LLA already describes that problem in that GitHub issue.  It's not relevant for SocketsHttpHandler.

Reply:
Will change comment.